freespace2/freespace.cpp

```diff
@@ -1983,7 +1983,6 @@ void game_init()
 
 	game_title_screen_close();
 
-	Script_system.RunInitFunctions();
```

Comment:
I think you removed the wrong one.

Reply:
I'm gonna come back to this after my meeting when I can focus

Reply:
Ok, I've done it correctly this time.

Reply:
Completely forgot to finish this review. Looks good now.